### PR TITLE
Fix Modal X button in Safari and Add style overrides for `NumberInput` Component

### PR DESCRIPTION
**Pull request by ChirashiSushi**
### Please review these reminders and either check the box or explain why not:

- [x] Read and followed the [contributing guidelines](https://eds.eks.dev.ethos-int.com/#/Guidelines?id=section-contribute)?
- [x] Component is exported from [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [x] Type is exported from [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [x] Bumped the yarn version?

### Screenshots and extra notes:
https://user-images.githubusercontent.com/84357906/176282490-dbb4226c-3827-45a5-8830-d5d98410c48a.mov

### CODE REVIEW

File: src/components/Tooltip/Tooltip.module.scss [hunk 1]
```diff
@@ -140,8 +139,8 @@
   }
 
   .closeButtonCircle {
-    width: var(--Space-12);
-    height: var(--Space-12);
+    width: var(--Space-32);
```
Comment (ChirashiSushi) [inline src/components/Tooltip/Tooltip.module.scss:142]: This was incorrectly set in the first place but somehow Chrome ignored the width and height limit and showed the full X button anyway. Safari doesn't want to cope with this error.

File: src/components/Tooltip/Tooltip.module.scss [hunk 2]
```diff
@@ -120,8 +120,7 @@
   width: calc(100vw - var(--Space-24));
   padding: var(--Space-64) var(--Space-24) var(--Space-80) var(--Space-24);
   max-height: 95vh;
-  overflow-y: scroll;
-
+  overflow-y: auto;
```
Comment (ChirashiSushi) [inline src/components/Tooltip/Tooltip.module.scss:123]: we don't need to show the vertical scroll all the time on mobile

File: src/components/NumberInput/NumberInput.js [hunk 3]
```diff
@@ -35,6 +35,9 @@ export const NumberInput = (props) => {
     autoComplete,
     maxLength,
     icon,
+    classOverrides,
```
Comment (ChirashiSushi) [inline src/components/NumberInput/NumberInput.js:38]: add these capabilities to `NumberInput` so we don't have to use custom CSS to overwrite. A follow up PR in CMS will use these properties for the `Age` question in EW.

**Review by WesleyDRobinson — APPROVED** [review]
lgtm 🚢

**Pull request merged**